### PR TITLE
Add handover documents

### PR DESCRIPTION
**Pull request by peteryates**
Add some handover documentation that should help whoever picks this project up get started.

### CODE REVIEW

File: docs/Heroku.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+There is currently no _actual_ data provided by anyone other than our seeds.
+To keep things simple, when the the application needs re-seeding we've torn
+down and re-seeded the database. This _isn't_ a long-term strategy, but while
+we have no users writing safe migrations is overkill.
+
+To completely reset and re-seed the database on Heroku, use these commands:
+
+```
+$ heroku pg:reset -a (your-app-name)
+$ heroku run SAFETY_ASSURED=1 DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rake db:schema:load db:seed -a (your-app-name)
```
Comment (rjlynch) [inline docs/Heroku.md:23]: Should probably add `heroku run -a (your-app-name) rails runner "Teacher.create! token: 'UUID'"` So they have a teacher to log in with after flattening the db. Also mention they'll want to check with the QA tester what UUID to use (we really need to stop running QA tests directly against staging)

**Review by rjlynch — CHANGES_REQUESTED** [review]
LGTM one minor note about creating a teacher

File: docs/Heroku.md (hunk 1 above)
Comment (peteryates) [reply docs/Heroku.md:23]: :+1:

**Review by rjlynch — APPROVED** [review]
Nice

**Pull request merged**